dask/bytes/tests/test_http.py

```diff
@@ -47,23 +47,25 @@ def test_simple(dir_server):
     f = open_files(root + fn)[0]
     with f as f:
         data = f.read()
-    assert data == open(os.path.join(dir_server, fn), "rb").read()
```

Comment:
@martindurant I was hoping that you could confirm that the changes in this file still test what the tests were intending to test.

Reply:
Yes, all is good. I would be very surprised if these changes actually helped though - but they certainly don't hurt.